packages/ava/src/text/schema/common.ts

```diff
@@ -0,0 +1,22 @@
+import type { CSSProperties } from 'react';
```

Comment:
Will we specify that `import type` needed? @pddpd

Reply:
🤔 I think `import type` is better than `import { AType }`

Reply:
ok, maybe we can add eslint ruls later

Reply:
TODO: eslint rules